CMSgov/easi-app
# [NOREF] Upgrade yarn dependencies based on dependabot PR

## Changes and Description

- Upgrade dependencies from https://github.com/CMSgov/easi-app/pull/2585 that I've tested and don't break compilation / cause test failures

### Identified as "Do not upgrade"
@apollo/client@3.10.4
@types/enzyme@3.10.18
formik@2.4.6
luxon@3.4.4
@types/luxon@3.4.2
react-table@7.8.0
@types/react-table@7.7.20
vite-ts-config-paths@4.3.2
@uswds/uswds@3.8.0
eslint-plugin-import@2.29.1
eslint-plugin-jsx-a11y@6.8.0
eslint-plugin-react@7.34.1

### Identified as "Safe to upgrade"
@cypress/code-coverage@3.12.39
@okta/okta-react@6.8.0
@okta/okta-signin-widget@7.18.1
@types/lodash@4.17.4
@types/no-scroll@2.1.2
@types/react-csv@1.1.10
@types/redux-mock-store@1.0.6
@types/redux-saga-routines@3.1.8
@types/text-encoding@0.0.39
detect-browser@5.3.0
dompurify@3.1.3
launchdarkly-react-client-sdk@3.2.0
react-hook-form@7.51.4
react-idle-timer@5.7.2
react-modal@3.16.1
@types/react-modal@3.16.3
react-select@5.8.0
react-to-print@2.15.1
redux-devtools-extension@2.13.9
redux-saga@1.3.0
sass@1.77.2
@babel/core@7.24.5
@babel/preset-env@7.24.5
@cypress/webpack-preprocessor@6.0.1
@vitejs/plugin-react@4.2.1
@wojtekmaj/enzyme-adapter-react-17@0.8.0
autoprefixer@10.4.19
cypress@13.9.0
eslint-plugin-react-hooks@4.6.2
regenerator-runtime@0.14.1
webpack@5.91.0

## How to test this change

- Run the app locally and ensure there's no functionality differences (visually or functionally)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
